packages/salesforce/configuration-schema.json

```diff
@@ -55,7 +55,6 @@
   "required": [
     "loginUrl",
     "username",
-    "password",
-    "securityToken"
```

Comment:
@mtuchi , is `securityToken` not required anymore?

Reply:
@taylordowns2000 according to jsforce, `securityToken` is optional - https://jsforce.github.io/document/#username-and-password-login

Reply:
ah wow. @aleksa-krolls , can you confirm that we don't _need_ users to provide their security token when creating a new Salesforce credential? (is this an admin setting in a client's salesforce system? something like, "require security token for API access"?)

Reply:
I think most salesforce instance are configured to require `securityToken` but if a client have a salesforce instance that does not require `securityToken` then the job should pass. 

The change i introduced also helps with credential creation on lighting which make the `Security Token` input optional